docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -243,15 +232,15 @@ server.3=zk-2.zk-headless.default.svc.cluster.local:2888:3888
 
 ### Achieving Consensus
 
-Consensus protocols require that the identifiers of each participant be 
-unique. No two participants in the Zab protocol should claim the same unique 
-identifier. This is necessary to allow the processes in the system to agree on 
-which processes have committed which data. If two Pods were launched with the 
-same ordinal, two ZooKeeper servers would both identify themselves as the same
- server.
+Consensus protocols require that the identifiers of each participant be
+unique. No two participants in the Zab protocol should claim the same unique
+identifier. This is necessary to allow the processes in the system to agree on
+which processes have committed which data. If Kubernetes launches two Pods with the
```

Comment:
"Kubernetes" isn't really the actor here -- again, reverting to passive voice is your friend here.